MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
+                                           HorizontalOptions="FillAndExpand"
+                                           VerticalOptions="FillAndExpand">
+                            <Label.Style>
+                                <Style TargetType="Label">
+                                    <Setter Property="FontAttributes" Value="Bold, Italic" />
+                                </Style>
+                            </Label.Style>
+                        </Label>
+                    </StackLayout>
+                </DataTemplate>
+       </sfgrid:SfDataGrid.GroupSummaryTemplate>
+
+   <sfgrid:SfDataGrid.GroupSummaryRows>
+            <sfgrid:DataGridSummaryRow ShowSummaryInRow="True">
+                <sfgrid:DataGridSummaryRow.SummaryColumns>
+                    <sfgrid:DataGridSummaryColumn Name="Salary" 
+                                                  MappingName="Salary" 
+                                                  Format="{}{Sum}"
+                                                  SummaryType="DoubleAggregate">
+                    </sfgrid:DataGridSummaryColumn>
+                    <sfgrid:DataGridSummaryColumn Name="customerID" 
+                                                  MappingName="CustomerID" 
+                                                  Format="{}{Count}"
+                                                  SummaryType="CountAggregate">
+                    </sfgrid:DataGridSummaryColumn>
+                </sfgrid:DataGridSummaryRow.SummaryColumns>
+            </sfgrid:DataGridSummaryRow>
+        </sfgrid:SfDataGrid.GroupSummaryRows>
+</sfgrid:SfDataGrid>
+
+{% endhighlight%}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# %}
+
+// To write a converter, follow the code example:
+
+public class GroupSummaryConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var data = value != null ? value as SummaryRecordEntry : null;
+        if (data != null)
+        {
+            SfDataGrid dataGrid = (SfDataGrid)parameter;
+            var summaryText = SummaryCreator.GetSummaryDisplayText(data,"Salary",dataGrid.View);
+            return "Total Salary:" + " " + summaryText.ToString();
+        }
+        return null;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return null;
+    }
+}
+
+
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with template loaded for entire group summary row](Images\Group-Summary\RowTemplate.png)
+
+N> The `DataTemplateSelector` can also be directly assigned to the `SfDataGrid.GroupSummaryTemplate`.
+
+### Displaying template for a column
+
+The template for a group summary column can be set by using `GridSummaryColumn.Template` to customize it based on requirement. 
+
+Refer the below code example in which a label is loaded in the template of group summary column.
```

Comment:
The template for a group summary column can be customized by using the GridSummaryColumn.Template property. This allows you to define a custom template according to your requirements.

Please refer to the code example below, which demonstrates how to load a label in the template of a group summary column:

Reply:
Modified.